streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -839,7 +839,8 @@ public class StreamsConfig extends AbstractConfig {
                     Type.STRING,
                     System.getProperty("java.io.tmpdir") + File.separator + "kafka-streams",
                     Importance.HIGH,
-                    STATE_DIR_DOC)
+                    STATE_DIR_DOC,
+                    "${java.io.tmpdir}")
```

Comment:
'define(java.lang.String, org.apache.kafka.common.config.ConfigDef.Type, java.lang.Object, org.apache.kafka.common.config.ConfigDef.Validator, org.apache.kafka.common.config.ConfigDef.Importance, java.lang.String)' in 'org.apache.kafka.common.config.ConfigDef' cannot be applied to '(java.lang.String, org.apache.kafka.common.config.ConfigDef.Type, java.lang.String, org.apache.kafka.common.config.ConfigDef.Importance, java.lang.String, java.lang.String)'

Reply:
Not sure what this comment is supposed to mean? Can you elaborate?

Reading between the lines, it sounds like a version mix-match, using an older version of `kafka-clients.jar` in combination with `kafka.streams.jar`?

Reply:
yes it's miss match , sorry for the false report